src/work-allocation-2/containers/task-assignment-confirm/task-assignment-confirm.component.spec.ts

```diff
@@ -151,22 +151,22 @@ describe('TaskAssignmentConfirmComponent', () => {
   });
 
   it('should redirect to the "All work" page on cancelling task assignment', () => {
-    window.history.pushState({returnUrl: 'all-work/tasks', showAssigneeColumn: false}, '', 'all-work/tasks');
+    mockSessionStorageService.getItem.and.returnValue('all-work/tasks');
```

Comment:
we could write a constant for 'all-work/tasks' given that it's used a number of times